frontend/src/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import classes from './NavBar.module.css';
+import pyramidIcon from '../assets/pyramidIcon.svg';
+
+export type NavBarProps = {
+  profileImage: string;
+}
+
+function NavBar(props: NavBarProps) {
+  return (
+    <div className={classes.container}>
+      <div className={classes.pyramids}>
+        <img src={pyramidIcon} className={classes.logo} ></img>
+        <h1 className={classes.title}>Pyramids</h1>
+      </div>
+      <div className={classes.pages}>
+        <h1>Timeline</h1>
```

Comment:
h1 isn't the most descriptive tag here - these aren't headings!
i'd suggest either `a` tags or `li` in a styled `ul`